packages/node/src/meta/health.service.ts

```diff
@@ -17,6 +18,13 @@ export class HealthService {
   private currentProcessingHeight?: number;
   private currentProcessingTimestamp?: number;
   private blockTime = 6000;
+  private defaultHealthTimeout = 960000;
+  private configHealthTimeout: number;
+
+  constructor(protected nodeConfig: NodeConfig) {
+    // health timeout always 1 mins longer than indexer timeout
```

Comment:
doesn't have to make it 1 min longer, the k8s will not terminate the pod for the first time health check fails.